deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -116,25 +119,41 @@ private void error(Element e, String msg, Object... args) {
     messager.printMessage(Diagnostic.Kind.ERROR, String.format(msg, args), e);
   }
 
-  private void generateRegistry(List<DeepLinkAnnotatedElement> elements) throws IOException {
+  private void generateDeepLinkLoader(List<DeepLinkAnnotatedElement> elements) throws IOException {
     MethodSpec.Builder loadMethod = MethodSpec.methodBuilder("load")
```

Comment:
Move this down right above the `for` loop on 132 to keep relevant localized by each other.